addons/aushelper/bootstrap.js

```diff
@@ -55,7 +57,7 @@ function startup() {
     wrk = Cc["@mozilla.org/windows-registry-key;1"].createInstance(Ci.nsIWindowsRegKey);
     wrk.open(wrk.ROOT_KEY_LOCAL_MACHINE, regCPUPath, wrk.ACCESS_READ);
   } catch (e) {
-    Cu.reportError("AUSHelper - unable to open registry. Exception: " + e);
+    Cu.reportError(`AUSHelper - unable to open registry. Exception: ${e}`);
```

Comment:
This is the [prefer-template rule](http://eslint.org/docs/rules/prefer-template) - it is new in es6, I didn't like it at first, but now I'm getting used to it, I find it a bit clearer than string concatenation.